bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/OppoBindingConstants.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.oppo.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link OppoBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Michael Lobstein - Initial contribution
+ */
+@NonNullByDefault
+public class OppoBindingConstants {
+    public static final String BINDING_ID = "oppo";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_PLAYER = new ThingTypeUID(BINDING_ID, "player");
+
+    public static final int MODEL83 = 83;
```

Comment:
Something to consider. You could put these different models in an enum, with specific model features in the enum, that are then accessed via generic methods. That way you can write the rest of the code in a generic way. It makes is also easier to add new models. If you understand what I mean. But that might be something if you want to do some generic improvements in the future.

Reply:
That is a good idea. But with this line of players, there will be no new models. I will keep your suggestion in mind for my other bindings.